### ISSUE TITLE
 @nrwl/gatsby:application with styles selected as none gives invalid imports

### ISSUE THREAD

**Issue by PsyGik** [nrwl/nx#4940]
## Current Behavior
 * Create a new workspace
 * Add a gatsby app using `nx g @nrwl/gatsby:application my-app --js`
 * Choose CSS to be `none`
 
It creates a styled component css and tries to import like so:
```
import React from 'react';
import styled from 'none';
import { ReactComponent as Logo } from './logo.svg';
import star from './star.svg';
const StyledApp = styled.div`
```

## Expected Behavior
No CSS framework should be used when selecting `none`

## Steps to Reproduce
* Create a new workspace
 * Add a gatsby app using `nx g @nrwl/gatsby:application my-app --js`
 * Choose CSS to be `none`
 
It creates a styled component css and tries to import like so:
```
// index.js
import React from 'react';
import styled from 'none'; // Hello Bug
import { ReactComponent as Logo } from './logo.svg';
import star from './star.svg';
const StyledApp = styled.div`
```
Repo: https://github.com/PsyGik/try-nx/blob/master/apps/try-gatsby/src/pages/index.js#L2

### Failure Logs

```
 ERROR #98124  WEBPACK

Generating development JavaScript bundle failed

Can't resolve 'none' in
'/Users/psygik/Development/temp/try-nx/trynx/apps/try-gatsby/src/pages'

If you're trying to use a package make sure that 'none' is installed. If you're trying to use
 a local file make sure that the path is correct.

File: src/pages/index.js

failed Building development bundle - 4.592s
```
### Environment

```
>  NX  Report complete - copy this into the issue template

  Node : 12.16.1
  OS   : darwin x64
  npm  : 6.13.4

  nx : Not Found
  @nrwl/angular : Not Found
  @nrwl/cli : 11.4.0
  @nrwl/cypress : 11.4.0
  @nrwl/devkit : 11.4.0
  @nrwl/eslint-plugin-nx : 11.4.0
  @nrwl/express : Not Found
  @nrwl/jest : 11.4.0
  @nrwl/linter : 11.4.0
  @nrwl/nest : Not Found
  @nrwl/next : Not Found
  @nrwl/node : 11.4.0
  @nrwl/react : 11.4.0
  @nrwl/schematics : Not Found
  @nrwl/tao : 11.4.0
  @nrwl/web : 11.4.0
  @nrwl/workspace : 11.4.0
  typescript : 4.0.7
```

**Comment by PsyGik** [comment]
https://github.com/nrwl/nx/blob/master/packages/gatsby/src/utils/styles.ts

Looks like the option `none` isn't supported?

**Comment by PsyGik** [comment]
Bump to keep active.. 📶

**Comment by xiongemi** [comment]
fixed in pr https://github.com/nrwl/nx/commit/f4ea0df which is merged to release 12.3.0
